framework/content/ecl-tables/ecl-tables.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Tables related behaviors.
+ */
+
+// eslint-disable-next-line import/prefer-default-export
+export function eclTables() {
```

Comment:
It may be an improvement.
Anyway, the code will have to be updated, as now it isn't optimal (too many DOM browsing that should be put in variables).
I'd say, we merge the branch as is, and we create a ticket for this, with your comment. If we have time to deal with it before thursday, great, in not we have at least something working.